addons/mail/tests/test_link_preview.py

```diff
@@ -86,3 +86,32 @@ def test_02_link_preview_create(self):
                     }
                 }]
             )
+
+    def test_get_link_preview_from_url(self):
+        preview_with_og_title = self.env['mail.link.preview']._get_link_preview_from_url('https://odoo.com')
```

Comment:
requests should be patched to not make actual requests to the outside world from tests.